Server/Communication/testhelper.py

```diff
@@ -148,6 +148,47 @@ def createUsersAndMatch(self, credentials1, team1, credentials2, team2):
 
 		return Game_User.objects.filter(user=Users.objects.filter(username=credentials1["username"])).first().game != None
 
+	def createUsersAndPlaceUnits(self, credentials1, team1, credentials2, team2):
+		self.assertTrue(self.createUsersAndMatch(credentials1, team1, credentials2, team2) != None)
```

Comment:
Shouldn't just check for != None here.  The return of the self.createUserAndMatch is a boolean normally and is only None if something weird happened and the database somehow wasn't updated and the code didn't throw an exception earlier.  Should just check that True is returned here.